### PR TITLE
Border reopening date

### PR DESCRIPTION
**Pull request by mirhamasala**
- Find the country reopening date
- Create `ReopeningDate` component
- Render `ReopeningDate` component
- Extract `badge` CSS styles into a CSS component

Screenshot 2020-07-30 at 1 29 32 PM
Screenshot 2020-07-30 at 1 29 43 PM

### CODE REVIEW

File: src/constants/travel.ts [hunk 1]
```diff
@@ -24,6 +24,11 @@ export const reopenings = Object.keys(countries)
 
 export const futureReopenings = reopenings.filter(({ date }) => Date.parse(date) > Date.now());
 
+export const countryReopeningDate = (countryCode: string) => {
```
Comment (yurikoval) [inline src/constants/travel.ts:27]: this is a search function, so let's use the same naming convention and group them together -> `findCountryReopeningDate`

File: src/assets/styles/components/_badge.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.badge {
```
Comment (yurikoval) [inline src/assets/styles/components/_badge.scss:1]: This looks like a component refactoring -- let's extract into generic badge component

Comment (mirhamasala) [reply src/assets/styles/components/_badge.scss:1]: Not sure about this, @yurikoval. Since we need this for the Worldmap tooltips.

Comment (yurikoval) [reply src/assets/styles/components/_badge.scss:1]: we would use the `Badge` component in that as well.

File: src/constants/travel.ts (hunk 1 above)
Comment (mirhamasala) [reply src/constants/travel.ts]: I did this one.

Let me know if you're okay with having the parsing logic here as well. Or if you prefer that I do it in the component.

```
export function findCountryReopeningDate(countryCode: string) {
  const date = Object.keys(countries[countryCode]?.reopening || []);
  return (!date.length ? false : Date.parse(date[0]));
}
```

File: src/assets/styles/components/_badge.scss (hunk 2 above)
Comment (mirhamasala) [reply src/assets/styles/components/_badge.scss:1]: If you want, we can get this merged, and I can do that in a separate PR next week.

**Pull request merged**